tests/test_cli.py

```diff
@@ -62,49 +62,219 @@ def make_args(project='test', username='test', password='test', **kwargs):
+             "2020-04-01 21:59:16", '-56.3', "ld2", "lid2", "lt2",
+             '18.7', "n2", "ru1", "ru2", "sc2", False, 'local', 'Local', 'lt1']
+        ]
+    }
+]
+
 
 class TestCli(unittest.TestCase):
 
     @mock.patch('commcare_export.cli._get_api_client', return_value=client)
     def test_cli(self, mock_client):
         args = make_args(
             query='tests/008_multiple-tables.xlsx',
+            output_format='json'
+        )
+        writer = JValueTableWriter()
+        with mock.patch('commcare_export.cli._get_writer', return_value=writer):
+            main_with_args(args)
+
+        for table in EXPECTED_MULTIPLE_TABLES_RESULTS:
+            assert writer.tables[table['name']] == table
+
+    @mock.patch('commcare_export.cli._get_api_client', return_value=client)
+    def test_cli_just_users(self, mock_client):
+        args = make_args(
             output_format='json',
+            users=True
         )
         writer = JValueTableWriter()
         with mock.patch('commcare_export.cli._get_writer', return_value=writer):
             main_with_args(args)
 
-        expected = [
-            {
-                "name": "Forms",
-                "headings": ["id", "name"],
-                "rows": [
-                    ["1", "f1"],
-                    ["2", "f2"]
-                ],
-            },
-            {
-                "name": "Other cases",
-                "headings": ["id"],
-                "rows": [
-                    ["case1"],
-                    ["case2"]
-                ],
-            },
-            {
-                "name": "Cases",
-                "headings": ["case_id"],
-                "rows": [
-                    ["c1"],
-                    ["c2"]
-                ],
-            }
-        ]
+        for table in EXPECTED_USERS_RESULTS:
+            assert writer.tables[table['name']] == table
+
+    @mock.patch('commcare_export.cli._get_api_client', return_value=client)
+    def test_cli_table_plus_users(self, mock_client):
+        args = make_args(
+            query='tests/008_multiple-tables.xlsx',
+            output_format='json',
+            users=True
+        )
+        writer = JValueTableWriter()
+        with mock.patch('commcare_export.cli._get_writer', return_value=writer):
+            main_with_args(args)
+
+        for table in EXPECTED_MULTIPLE_TABLES_RESULTS + EXPECTED_USERS_RESULTS:
+            assert writer.tables[table['name']] == table
+
+    @mock.patch('commcare_export.cli._get_api_client', return_value=client)
+    def test_cli_just_locations(self, mock_client):
+        args = make_args(
+            output_format='json',
+            locations=True
+        )
+        writer = JValueTableWriter()
+        with mock.patch('commcare_export.cli._get_writer', return_value=writer):
+            main_with_args(args)
+
+        for table in EXPECTED_LOCATIONS_RESULTS:
+            assert writer.tables[table['name']] == table
```

Comment:
These lines are common to all tests and could be extracted to a util function:
```suggestion
        self._test_cli(args, EXPECTED_LOCATIONS_RESULTS)
```

Reply:
Done.